examples/basic2/go.mod

```diff
@@ -2,7 +2,7 @@ module basic2
 
 go 1.15
 
-// replace github.com/status-im/go-waku => ../..
+replace github.com/status-im/go-waku => ../..
```

Comment:
That was a mistake to be commented right?

Reply:
yeah!